src/dbgshim/dbgshim.cpp

```diff
@@ -2151,3 +2156,41 @@ CLRCreateInstance(
 
     return pDebuggingImpl->QueryInterface(riid, ppInterface);
 }
+
+HRESULT CreateCoreDbgRemotePort(HMODULE hDBIModule, DWORD portId, LPCSTR assemblyBasePath, IUnknown **ppCordb)
+{
+    HRESULT hr = S_OK;
+
+    FPCoreCLRCreateCordbObjectRemotePort fpCreate =
+        (FPCoreCLRCreateCordbObjectRemotePort)GetProcAddress(hDBIModule, "CoreCLRCreateCordbObject");
```

Comment:
Please don't use the same name as coreclr with a different signature.  Something like "CreateRemoteCordbObject" or something.

Reply:
done :)